manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -230,6 +230,12 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: spec.nodeName
+          {{- if eq .Values.meshConfig.compliancePolicy "fips-140-2" }}
+          - name: ISTIO_META_FIPS
+            value: "true"
+          - name: GRPC_SSL_CIPHER_SUITES
+            value: ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384
```

Comment:
you donot have to explicitly set this, make these values as default. So donot need to change so many yamls

Reply:
This is read by GRPC C core - it has to be an environment variable. The value is different for FIPS and non-FIPS.

Reply:
Can we simply make `GRPC_SSL_CIPHER_SUITES` a hardcoded env var in the container dockerfile itself that depends on the presence of `ISTIO_META_FIPS`?

Or perhaps we should actually hardcode `GRPC_SSL_CIPHER_SUITES` in our code via a startup `os.Setenv`?

(Or maybe we just ship FIPS image variants that have this baked in)

I also agree it's a little gross to put this in every single Helm YAML, especially since it's a nonconfigurable list of values.

Reply:
Or just in `func (e *envoy) Run(abort <-chan error) error {` based on ISTIO_META_FIPS

Reply:
Added in the Run function.